compiler/noirc_evaluator/src/ssa/opt/loop_invariant.rs

```diff
@@ -804,4 +827,32 @@ mod test {
         let ssa = ssa.loop_invariant_code_motion();
         assert_normalized_ssa_equals(ssa, expected);
     }
+
+    #[test]
+    fn do_not_transform_unsafe_sub_to_unchecked() {
```

Comment:
Should a similar test be added where `sub` is transformed to `unchecked_sub`?

Reply:
I will add one

Reply:
done